distribution/ddf/pom.xml

```diff
@@ -80,25 +80,6 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-dependency-plugin</artifactId>
                 <executions>
```

Comment:
Can the dependency on lines 237-242 also be removed from this file?

Reply:
good catch: fixed